event-handler/teleport_events_watcher.go

```diff
@@ -193,13 +193,16 @@ func (t *TeleportEventsWatcher) fetch(ctx context.Context) error {
 	pos := 0
 
 	// Convert batch to TeleportEvent
-	for i, e := range b {
+	for _, e := range b {
+		if _, ok := t.config.SkipEventTypes[e.Type]; ok {
+			continue
```

Comment:
Is it worth emitting a debug level message that an event was skipped due to the flag?